README

```diff
@@ -1,43 +1,92 @@
-
+--------
+opa-hfi1
+--------
+This is the source version of the hfi1 driver that is released with official IFS
+builds. However it does at times, depending on the branch contain additional
+fixes that could not make the IFS release that customers may find useful.
+
+Branches are generally named based on IFS release. For instance:
+opa-10_11_0_1 represents IFS release 10.11.0.1
+
+At a high level building this software is comprised of the following steps:
+
+1. Build source RPM (.srpm) file
+2. Build a binary RPM (.rpm) from the .srpm
+3. Install RPM or extract .ko files using rpm2cpio
+4. Remove any running hfi1 driver
+5. insmod/modprobe the .ko file(s) from step 3
+
```

Comment:
This outline is still a bit confusing to me. What I think is being described later is:
```
1. Build source RPM (.srpm) file
2. Build a binary RPM (.rpm) from the .srpm
3. Activate the new driver by either:
    A. Install RPM: install the binary rpm and reboot
    B. Manually insmod: extract .ko files using rpm2cpio, rmmod/insmod driver
4. Verify that the correct driver is running.
```
And then rely on the details in those sections to help the user make the choice for #3.